mail.py

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
This commit messages is not clear, I believe that you want to say:
`Changing email script to support multiple emails subjects in inbox` or something like.